# This is the message format for the battery monitor for the AAUSHIP
float32[4] bank1
float32[4] bank2
